libsql-server/src/stats.rs

```diff
@@ -82,6 +177,12 @@ pub struct Stats {
     slowest_queries: Arc<RwLock<BTreeSet<SlowestQuery>>>,
     #[serde(default)]
     embedded_replica_frames_replicated: AtomicU64,
+    #[serde(default)]
+    query_count: AtomicU64,
+    #[serde(default)]
+    query_latency: AtomicU64,
+    #[serde(skip)]
+    queries: Arc<RwLock<QueriesStats>>,
```

Comment:
I would consider using `parking_lot::RwLock` or Mutex here instead of the std one.

Reply:
I tried replacing all stats `RwLocks` with `parking_lot::RwLock` but it breaks (de)serialization of the stats struct.
Let me know if you want me to change just this single lock, since we skip serialization of the `queries` field.